Pipfile

```diff
@@ -57,6 +56,8 @@ psycopg2 = "==2.7.5"
 google-cloud-storage = "==1.14.0"
 docker = "==3.7.2"
 django-bitfield = "==1.9.3"
+django-sslserver = "*"
```

Comment:
Shouldn't these be under the dev dependencies heading?

Reply:
You are right!